src/Analyzer.Cli/CommandLineParser.cs

```diff
@@ -41,10 +41,7 @@ private RootCommand SetupCommandLineAPI()
             rootCommand = new RootCommand();
             rootCommand.Description = "Analyze Azure Resource Manager (ARM) Templates for security and best practice issues.";
 
-            // It has two commands - analyze-template and analyze-directory
             rootCommand.AddCommand(SetupAnalyzeTemplateCommand());
```

Comment:
Can you update this too, so instead of adding a new command, it just adds options directly to the root command?